kafka-rest/src/main/java/io/confluent/kafkarest/controllers/ControllersModule.java

```diff
@@ -24,6 +24,7 @@
 
   protected void configure() {
     bind(BrokerManagerImpl.class).to(BrokerManager.class);
+    bind(BrokerConfigManagerImpl.class).to(BrokerConfigManager.class);
```

Comment:
Move this line above the `BrokerManager` line to keep them lexicographically sorted.